functests/1_performance/netqueues.go

```diff
@@ -67,6 +71,14 @@ var _ = Describe("[ref_id: 40307][pao]Resizing Network Queues", func() {
 			By("Updating the performance profile")
 			profiles.UpdateWithRetry(profile)
 		}
+		// Verify the tuned profile is updated on the worker-cnf nodes
```

Comment:
I'd actually use a `Eventually` block. `Sleep`s are always suspicious (and yes, I'm guilty of using them sometimes. So let's stop :) )

Reply:
this is actually a bit tricky here, I would even remove the check of the tuned profile now that I think of it since it can remain from test to test and the sleep is what does the job.
since the tuned profile will have the same name in every test there will have to be another way to check if the update was applied (will need to think about this further)